helper/include/odp/helper/strcpy.h

```diff
@@ -0,0 +1,62 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ * Copyright (c) 2024 Nokia
+ */
+
+/**
+ * @file
+ *
+ * ODP string copy helper
+ */
+
+#ifndef ODPH_STRCPY_H_
+#define ODPH_STRCPY_H_
+
+#include <odp/api/hints.h>
+#include <string.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @defgroup odph_strcpy ODPH strcpy
```

Comment:
A more generic module name could be useful if we anticipate other similar functions being added later on.